src/QtAVPlayer/qavplayer.cpp

```diff
@@ -260,6 +260,7 @@ void QAVPlayerPrivate::doDemux()
                 qWarning() << "Could not seek:" << err_str(ret);
             }
             pendingPosition = -1;
+            setMediaStatus(QAVPlayer::LoadedMedia);
```

Comment:
it is called on demuxer thread, but setMediaStatus should be on main thread.
Use
call([this]{ setMediaStatus(QAVPlayer::LoadedMedia); }